### PR TITLE
Bp 359 Query interface - notification for successfully saving event type

### PR DESCRIPTION
**Pull request by friedow**

### CODE REVIEW

File: src/Notification/Notification.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
 import React, {Component} from 'react';
 class Notification extends Component {
+
+    componentWillReceiveProps(nextProps) {
```
Comment (hpibptlab) [inline src/Notification/Notification.js]: MAJOR Remove the unused function parameter "nextProps". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/__tests__/ProductView/QueryInterface.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ test('Handling save query', () => {
     expect(callbackMock).toBeCalled();
 });
 
-test('Handling save query failure', () => {
+test('Handling save query sucess', () => {
```
Comment (tommartensen) [inline src/__tests__/ProductView/QueryInterface.js]: sucess -> success

File: src/__tests__/ProductView/__snapshots__/QueryInterface.js.snap [hunk 3]
```diff
@@ -568,7 +568,7 @@ exports[`test Handling change query 2`] = `
 </div>
 `;
 
-exports[`test Handling save query failure 1`] = `
+exports[`test Handling save query sucess 1`] = `
```
Comment (tommartensen) [inline src/__tests__/ProductView/__snapshots__/QueryInterface.js.snap]: sucess -> success

**Review by tommartensen — CHANGES_REQUESTED** [review]
Please fix the two typos and see the other comment I will write directly into the PR

**Comment by tommartensen** [comment]
@friedow: The implementation just works once on every page. 
If I saved an event type, the notification slides in. If I re-open the query interface and save a new event type, the notification does not appear again (neither an error).
Sometimes the notification does not work at all.

**Review by tommartensen — APPROVED** [review]

**Pull request merged**